test/operators/test_fermionic_op.py

```diff
@@ -166,6 +166,9 @@ def test_reduce(self):
         reduced_op = fer_op.reduce()
         self.assertSetEqual(frozenset(reduced_op.to_list()), frozenset([("N", 2), ("E", 1)]))
 
+        fer_op = FermionicOp(("+", 1)) + FermionicOp(("-", 1j)) + FermionicOp(("+", 1j))
```

Comment:
Ideally the unit tests, where they have multiple asserts, where testing will stop on the first one that fails, rather than completing all the parts of that test, which may be useful to narrow down any problem, that these should either be parameterized by `ddt` or use the `with self.subTest` construct to separate the parts if the test is not so easily parameterized or makes sense say to have different unit tests

Reply:
Yes, ddt is better. Other unit tests in this file should be revised with ddt maybe in another PR.